Week1/Assignment_blog/index.html

```diff
@@ -0,0 +1,352 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>예원 블로그</title>
+		<link rel="stylesheet" href="style.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css" />
+	</head>
+	<body>
+		<header>
+			<i class="fa-solid fa-blog"></i>
+			<h1>얘옹 블로그</h1>
+			<i class="fa-solid fa-bars"></i>
+		</header>
+		<div class="container">
+			<nav class="tabs">
+				<ul class="navContainer">
+					<li><a href="#section-all">전체</a></butt>
+					<li><a href="#section-daily">일상</a></li>
+					<li><a href="#section-dev">개발</a></li>
+				</ul>
+			</nav>
+
+			<main>
+				<section>
+					<h3>예원 블로그에 오신 것을 환영합니다</h3>
+					<div class="homeSection">
+						<div class="item item1"><img src="./images/home1.png" /></div>
+						<div class="item item2"><img src="./images/home2.png" /></div>
+						<div class="item item3"><img src="./images/home3.png" /></div>
+						<div class="item item4"><img src="./images/home4.png" /></div>
+						<div class="item item5"><img src="./images/home5.png" /></div>
+					</div>
+				</section>
+				<section id="section-all">
+					<h3>전체</h3>
+					<div class="grid">
+						<article>
+							<img src="./images/images.jpeg" />
+							<h5>박명수젤조아</h5>
+							<span class="heart"></span>
+							<h6>좋아요 12개</h6>
+						</article>
+						<article>
+							<img src="./images/images.jpeg" />
+							<h5>박명수젤조아</h5>
+							<span class="heart"></span>
```

Comment:
```suggestion
							<span class="heart" />
```
태그를 바로 닫아줘도 동일하게 작동하지 않나요?